src/components/dockerRegistry/Docker.tsx

```diff
@@ -713,21 +752,21 @@ function DockerForm({
     }
 
     const handleOCIRegistryStorageAction = (key) => {
-        if (key === OCIRegistryConfigConstants.PUSH) {
```

Comment:
Why did we remove the constants?

Reply:
Earlier it was unrelevant. Added relevant constant